packages/sonarwhal/tests/lib/config.ts

```diff
@@ -270,3 +270,37 @@ test(`if a Rule has an invalid configuration, it should tell which ones are inva
 
     t.is(invalid.length, 1);
 });
+
+test('If formatter is specified as CLI argument, fromConfig method will use that to build SonarwhalConfig', (t) => {
+    const { config } = t.context;
+    const userConfig = {
+        connector: { name: 'chrome' },
+        formatters: ['summary', 'excel'],
+        rules: { 'apple-touch-icons': 'warning' }
+    } as UserConfig;
+    const cliOptions = { _: ['https://bing.com'], formatters: 'database' } as CLIOptions;
+
+    const result = config.SonarwhalConfig.fromConfig(userConfig, cliOptions);
+
+    t.true(result.formatters.length===1);
```

Comment:
@kshyju Can you change the `t.true` to [`t.is(value, expectedValue)`](https://github.com/avajs/ava/tree/1e0e8e84dbc8ed70286d01dd9c8d290f30533aad#isvalue-expected-message) just so that the code is consistent  with what we use in general. Thanks!

Reply:
Sorry! Fixed now.